cms/djangoapps/contentstore/xblock_storage_handlers/xblock_helpers.py

```diff
@@ -13,3 +14,13 @@ def usage_key_with_run(usage_key_string):
     usage_key = UsageKey.from_string(usage_key_string)
     usage_key = usage_key.replace(course_key=modulestore().fill_in_run(usage_key.course_key))
     return usage_key
+
+
+def get_children_tags_count(xblock):
+    """
+    Returns a map with tag count of each child
+    """
+    children = xblock.get_children()
+    child_usage_keys = [str(child.location) for child in children]
+    tags_count_query = ','.join(child_usage_keys)
+    return get_object_tag_counts(tags_count_query, count_implicit=True)
```

Comment:
There seems to be double counting happening with the implicit tags. If there are multiple 'leaf' tags, the implicit ancestors are being counted twice showing an inflated tag count. Example, it should show 19 instead of 21:

<img width="648" alt="Screen Shot 2023-12-17 at 12 53 49 PM" src="https://github.com/openedx/edx-platform/assets/6829768/82f27a27-c4ba-4b7a-b243-08c9af61db19">

Reply:
FYI @bradenmacdonald

Reply:
@yusuf-musleh  @ChrisChV  I fixed it: https://github.com/openedx/openedx-learning/pull/133 Could one of you please test and approve that PR? Then I'll ask another CC to merge it since Jill is away.

Reply:
@bradenmacdonald Done, it's working as expected now, left my approval.